rnaget-openapi.yaml

```diff
@@ -818,66 +1220,91 @@ components:
       properties:
```

Comment:
the `study` schema should have a `description`

Reply:
added